api/routes/billing.py

```diff
@@ -586,3 +594,21 @@ async def get_cost_by_sample(
     billing_layer = _get_billing_layer_from(author)
     records = await billing_layer.get_cost_by_sample(connection, query)
     return [BillingTotalCostRecord.from_json(record) for record in records]
+
+
+@router.get('/group-info', operation_id='getBillingGroupInfo')
+@alru_cache(ttl=BILLING_CACHE_RESPONSE_TTL)
+async def get_billing_group_info() -> list[BillingTeamRecord]:
+    """
+    Get billing groups and topics related to CPG teams
+    """
+    name = (
+        f'projects/{METAMIST_GCP_PROJECT}/secrets/{BILLING_GROUP_INFO}/versions/latest'
+    )
+    response = secretmanager.SecretManagerServiceClient().access_secret_version(
+        name=name
+    )
+    return [
```

Comment:
See below for comment re using pydantic's field aliases. Once that is done you can change this to.

```python
return TypeAdapter(list[BillingTeamRecord]).validate_json(response.payload.data.decode('UTF-8'))
```

also adding the import for `from pydantic import TypeAdapter`

https://docs.pydantic.dev/latest/concepts/type_adapter/